lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -154,6 +154,10 @@ def call(params)
                 filters << row.to_hash
               end
             end
+            if filters.empty?
+              fail 'The filter set can not be empty when using sync_multiple_projects_* mode as the filters contain \
```

Comment:
Calls 'fail 'the filter set can not be empty when using sync_multiple_projects_* mode as the filters contain \; the project ids in which the permissions should be changed'' 2 times (DuplicateMethodCall)